package.json

```diff
@@ -5,7 +5,8 @@
   "private": true,
   "scripts": {
     "build": "webpack -p",
-    "watch": "concurrently --kill-others \"webpack -w\" \"node test-server.js\""
+    "watch": "concurrently --kill-others \"webpack -w\" \"node test-server.js\"",
+    "generate-icons": "vsvg -s ./svg-icons -t ./src/compiled-icons"
```

Comment:
Why do you check-in the source and generated icons, when you add a generate step?